create-doi-request/build.gradle

```diff
@@ -11,11 +11,7 @@ dependencies {
     compileOnly project(":publication-commons")
     testImplementation project(":publication-commons")
 
-    compileOnly group: 'com.github.BIBSYSDEV.nva-commons', name: 'core', version: project.ext.nvaCommonsVersion
-    testImplementation group: 'com.github.BIBSYSDEV.nva-commons', name: 'core', version: project.ext.nvaCommonsVersion
-
-    compileOnly group: 'com.github.BIBSYSDEV.nva-commons', name: 'apigateway', version: project.ext.nvaCommonsVersion
-    testImplementation group: 'com.github.BIBSYSDEV.nva-commons', name: 'apigateway', version: project.ext.nvaCommonsVersion
+    implementation group: 'no.unit.nva', name: 'apigateway', version: project.ext.nvaCommonsVersion
```

Comment:
simplify the dependencies by using implementation. When this will be a problem we will deal with it